tools/base/fpgaconf/fpgaconf.c

```diff
@@ -123,7 +126,7 @@ void help(void)
 	       "        fpgaconf [-hvn] [-B <bus>] [-D <device>] [-F <function>] [-S <socket-id>] <gbs>\n"
 	       "\n"
 	       "                -h,--help           Print this help\n"
-	       "                -v,--verbose        Increase verbosity\n"
+	       "                -V,--verbose        Increase verbosity\n"
 	       "                -n,--dry-run        Don't actually perform actions\n"
```

Comment:
Do you want to add a line for -v for version ?

Reply:
Doh, good catch! https://github.com/OPAE/opae-sdk/pull/1361